app/views/courses/CoursesNotAssignedModal.coffee

```diff
@@ -0,0 +1,9 @@
+ModalView = require 'views/core/ModalView'
+template = require 'templates/courses/courses-not-assigned-modal'
+
+module.exports = class CoursesNotAssignedModal extends ModalView
+  id: 'courses-not-assigned-modal'
+  template: template
+
+  initialize: (options) ->
+    _.assign(@, _.pick(options, 'selected', 'totalSpotsAvailable', 'unenrolledStudents'))
```

Comment:
We do this pattern a lot of sticking template-accessible variables on `view`, but IMHO I think it'd be more clear if we did things like this:

```
  initialize: (@options) ->

  getRenderData: ->
    _.merge super(), _.pick(@options, 'selected', 'totalSpotsAvailable', 'unenrolledStudents')
```

and then use `selected`, etc directly in the view.

This is more a comment on the pattern we have than your particular code, which does follow the usual pattern.